src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -1,14 +1,14 @@
 import GraphContainer from '@gitkraken/gitkraken-components';
 import type {
 	GraphColumnSetting,
+	GraphColumnsSettings,
 	GraphContainerProps,
 	GraphPlatform,
 	GraphRef,
 	GraphRefGroup,
 	GraphRefOptData,
 	GraphRow,
-	OnFormatCommitDateTime,
-} from '@gitkraken/gitkraken-components';
+ OnFormatCommitDateTime } from '@gitkraken/gitkraken-components';
```

Comment:
Formatting issue here

Reply:
Fixed in commit: `Fixed formatting issue`.